src/components/L2/Dropdowns/AccountsDropdown.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import * as ob from 'urbit-ob';
+import { Icon, Row } from '@tlon/indigo-react';
+import { Just } from 'folktale/maybe';
+
+import Sigil from 'components/Sigil';
+import LayerIndicator from 'components/L2/LayerIndicator';
+import { useWallet } from 'store/wallet';
+import { useHistory } from 'store/history';
+import { usePointCursor } from 'store/pointCursor';
+import { usePointCache } from 'store/pointCache';
+import { clearInvitesStorage } from 'store/storage/roller';
+import CopiableAddressWrap from 'components/copiable/CopiableAddressWrap';
+import Dropdown from './Dropdown';
+
+import './AccountsDropdown.scss';
+
+const AccountsDropdown = () => {
+  const [open, setOpen] = useState<boolean>(false);
+  const walletInfo: any = useWallet();
+  const { push, names, reset }: any = useHistory();
+  const { setPointCursor }: any = usePointCursor();
+  const { controlledPoints }: any = usePointCache();
+
+  const canBitcoin = Just.hasInstance(walletInfo.urbitWallet);
+
+  const points =
+    controlledPoints?.value?.value?.ownedPoints?.map((point: string) =>
+      Number(point)
+    ) || [];
+
+  const currentAddress = walletInfo?.wallet?.value?.address || '';
+  const addressLeadingFive = currentAddress.slice(0, 5);
+  const displayAddress = `${addressLeadingFive}...${currentAddress.slice(-4)}`;
```

Comment:
If it suits the needs of this use case, how about swapping this out for `abbreviateAddress` in `src/lib/utils/address.ts`? Otherwise, this is fine